index.html

```diff
@@ -42,5 +43,185 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
       </ul>
     </div>
   </section>
+
+<!-- Section: Recent Works -->
+<section class="works_main">
+
+  <div class="works_title">
+    <h3 class="Title-work-main Title2_dark">My recent Works</h3>
+    <span class="line_div"></span>
+  </div>
+
+  <div class="main_card_work">
+    <div class="main_card">
+      <div class="cnt_img_main_card">
+        <img class="work_img" src="/images/card1_img.png" alt="Project image">
+      </div>
+      <div class="cnt_info_main_card">
+        <h3 class="main_work Title2_dark">Multi-Post Stories</h3>
```

Comment:
Please consider using the same styles as used in the Figma Design to meet the project requirements. Consider focusing on the `font-weight, font-size, line-height, font-family, width, and height properties` and others to meet the Figma template.

## Remember:- 

You need to make the exact copy of the design which is the goal of this assignment. Moreover, it's also gonna be our future portfolio so consider taking close attention to each style rather than using your own custom set of styles which you like.

## Your Design:

![image](https://user-images.githubusercontent.com/94223367/227763343-e66208e6-5a86-43d6-a1bb-2533735f8790.png)


## Figma Design:

![image](https://user-images.githubusercontent.com/94223367/227763340-144cd9d7-2d74-4c63-b194-eec5eac4b6ed.png)